helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -45,11 +45,17 @@ data:
       -Dsolr.port={{ index .Values "alfresco-search" "external" "port" }}
       {{- end }}
       {{- end }}
+      -Dtransform.service.url=http://{{ template "alfresco.shortname" . }}-router
       -Dalfresco-pdf-renderer.url=http://{{ template "content-services.shortname" . }}-pdfrenderer
       -Dimg.url=http://{{ template "content-services.shortname" . }}-imagemagick
       -Djodconverter.url=http://{{ template "content-services.shortname" . }}-libreoffice
       -Dtika.url=http://{{ template "content-services.shortname" . }}-tika
-      -Dtransform.misc.url=http://{{ template "content-services.shortname" . }}-transform.misc
+      -Dtransform.misc.url=http://{{ template "content-services.shortname" . }}-transform-misc
```

Comment:
Can we please have the new -DlocalTransformer... lines between the T-Router entry on line 48 and the legacy entries on 49. Logically that is the order.